### ISSUE TITLE
Author very basic tmLanguage file for syntax highlighting ADL code in VS Code

### ISSUE THREAD

**Issue by daviwil** [Azure/adl#222]
This will be necessary to provide a baseline authoring experience.

**Comment by nguerrera** [comment]
You've probably seen it, but just for the record on this issue, there's a start at this in https://github.com/Azure/adl/pull/185
 
The grammar has diverged from that, though.

**Comment by daviwil** [comment]
Yep, I did see it!  Definitely keeping it around so that we can use it as a starting point.  I couldn't tell if it was based on the TypeScript syntax file or if it was hand-authored for the New ADL.

**Comment by nguerrera** [comment]
It's based on an initial proposal for our own language (so not the original, original one that was a TypeScript subset), but the language as parsed now has drifted away from that grammar too.
